src/dicomweb_client/cli.py

```diff
@@ -58,10 +58,18 @@ def _get_parser():
         '--cert', dest='cert', metavar='CERT-FILE',
         help='path to a client certificate file in PEM format'
     )
+    parser.add_argument(
+        '-t', '--token', dest='token', metavar='TOKEN',
```

Comment:
Do we want to be more specific and differentiate between different kinds of tokens (i.e., ``--bearer-token`` instead of ``--token``)?

Reply:
It might be nice to call out `--bearer-token` explicitly. Any other token could be registered with a generic key/value for appending a header?

IE. I have a good idea what the behavior when providing `--bearer-token`. I would be unsure of the result when providing `--token`

Reply:
Yes, --bearer-token makes more sense.

I was wondering if we shouldn't just have a generic --header option as well and allow multiple --header flags on the command line.  Perhaps even just copy the behavior.

https://curl.haxx.se/docs/manpage.html#-H

Reply:
I like the idea of a generic ``--header`` option. This could be implemented similar to the existing ``--filter`` argument: https://github.com/MGHComputationalPathology/dicomweb-client/blob/731864c88e2e31d6a6eb4c34ccbacd62c8b7eba8/src/dicomweb_client/cli.py#L103-L107